net-ftp/yafc/yafc-1.3.7-r2.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools bash-completion-r1
+
+DESCRIPTION="Console ftp client with a lot of nifty features"
+HOMEPAGE="https://github.com/sebastinas/yafc"
+SRC_URI="https://github.com/sebastinas/yafc/archive/v${PV}.tar.gz"
```

Comment:
I'd suggest to use `SRC_URI="https://github.com/sebastinas/yafc/archive/v${PV}.tar.gz -> ${P}.tar.gz"` instead

Reply:
Because this is how the files get saved in distfiles and `v2.0.1.tar.gz` says nothing?